source/plugins/go/src/utils_windows.go

```diff
@@ -26,6 +27,25 @@ func CreateWindowsNamedPipesClient(namedPipe string) {
 		Log("Error::AMA::Unable to open Named Pipe %s", err.Error())
 	} else {
 		Log("Windows Named Pipe opened without any errors")
-		ContainerLogNamedPipe = conn
+		*namedPipeConnection = conn
 	}
 }
+
+func CreateGenevaOr3PNamedPipe(namedPipeConnection *net.Conn, datatype string, errorCount *float64, isGenevaLogsIntegrationEnabled bool, refreshTracker *time.Time) bool {
```

Comment:
implementation doesnt match with the function and we should have better function name to match with implemention.

Reply:
Yes makes sense. First I was always creating a named pipe but then I thought of checking here as well.